src/index.js

```diff
@@ -48,7 +48,8 @@ const defaultOptions = {
       headers: void 0,
       mode: 'cors',
       cache: void 0
-    }
+    },
+    uglify: null
```

Comment:
Let's rename it to minify please and would be good if you could add tests for that + update README/FAQ with new information.